app/src/main/python/gogdl/api.py

```
@@ -0,0 +1,118 @@
+import gogdl.constants as constants
+
+
+class ApiHandler:
+    def __init__(self, auth_manager):
+        self.auth_manager = auth_manager
+        self.logger = logging.getLogger("API")
+        self.session = requests.Session()
+        adapter = requests.adapters.HTTPAdapter(pool_maxsize=cpu_count())
+        self.session.mount("https://", adapter)
+        self.session.headers = {
+            'User-Agent': f'gogdl/1.0.0 (Android GameNative)'
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)